polkadot/node/network/protocol/src/peer_set.rs

```diff
@@ -227,6 +241,8 @@ pub enum CollationVersion {
 	V1 = 1,
 	/// The second version.
 	V2 = 2,
+	/// Same format as V2,
+	VStaging = 3,
```

Comment:
Then why have it? collations have their own peer set, why do we need that?

Reply:
See bellow,